libs/blocks/graybox/graybox.css

```diff
@@ -0,0 +1,283 @@
+:root {
+  --gb-container-bg: white;
+  --gb-outline-color: #1473E6;
+  --gb-container-btn-bg: #505050;
+  --gb-overlay-color: rgba(0 0 0 / 45%);
+  --gb-modal-bg: #323232;
+  --gb-container-border: 4px solid var(--gb-outline-color);
+  --gb-page-outline-border: 8px solid var(--gb-outline-color);
+  --gb-toggle-arrow: 3px solid white;
+}
+
+.graybox {
+  display: none;
+}
+
+.gb-graybox-body {
+  position: relative;
+}
+
+.gb-graybox-body:not(.gb-no-border)::before {
+  content: '';
+  position: absolute;
+  top: calc(50% + 8px);
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: calc(100% - 15.5px);
+  height: 100%;
+  border: var(--gb-page-outline-border);
+  z-index: 3000;
```

Comment:
I'm a little worried about just using an exceedingly high z-index, could we have some root variables in the sense of:
```
  --level1: 1;
  --level2: 2;
  --level3: 10;
```

That would help in terms of maintainability and predicitability